City-of-Helsinki/drupal-helfi
# UHF-8 HDBT Admin theme

To test:

To make this change possible a new repository was set up called [drupal-hdbt-admin](https://github.com/City-of-Helsinki/drupal-hdbt-admin) and it was added to the [drupal-helfi-platform](https://github.com/City-of-Helsinki/drupal-helfi-platform) and [drupal-repository](https://github.com/City-of-Helsinki/drupal-repository).

NOTICE: The helfi_tpr module was renamed so I would recommend reinstall of local setup.

1. make new
2. The HDBT Admin theme should be the admin theme of the site and HDBT theme should be the default theme otherwise.
3. The themes should now be loaded to public/themes/custom folder and be in the git ignore so they will not appear to listed changes.
4. Also the custom modules starting with the pattern helfi_ should be ignored from the version control.
5. Make sure that the hdbt and hdbt_admin themes are also cloned so that they have their own .git folders.
